great_expectations/expectations/metrics/column_map_metrics/column_values_not_in_set.py

```diff
@@ -14,6 +15,14 @@
 )
 from great_expectations.expectations.metrics.util import parse_value_set
 
+try:
+    import polars as pl
+    from polars import DataFrame as pl_DataFrame
+
+except ImportError:
+    pl_DataFrame = None
+    pl = None
+
```

Comment:
If possible let's use the import manager here too.

Reply:
done :)